pyvista/plotting/qt_plotting.py

```diff
@@ -537,6 +537,11 @@ def remove_actor(self, actor, reset_camera=None):
         """Remove an actor."""
         self.signal_remove_actor.emit(actor)
 
+    def render(self):
+        """Render the main window."""
+        if hasattr(self, 'ren_win'):
+            self.ren_win.Render()
```

Comment:
is this overriding and inherited method? Currently, if I call `render()`, an error from Qt is thrown

Reply:
No it's not supposed to be overriding. It's the symmetric of `Plotter`'s `render`. I don't know why you have an error here. 
You call `render` through `BackgroundPlotter`?